entity-framework/core/what-is-new/ef-core-5.0/plan.md

```diff
@@ -0,0 +1,227 @@
+
+### Supported platforms
+
+EF Core 5.0 is planned to run on any .NET Core 5.0 platform based on the [convergence of these platforms to .NET Core](https://devblogs.microsoft.com/dotnet/introducing-net-5/). What this means in terms of .NET Standard and the actual TFM used is still TBD.
+
+EF Core 5.0 will not run on .NET Framework.
+
+### Breaking changes
+
+EF Core 5.0 will contain some breaking changes, but these will be much less severe than was the case for EF Core 3.0. Our goal is to allow the vast majority of applications to update without breaking.
+
+It is expected that there will be some breaking changes for database providers, especially around TPT support. However, we expect the work to update a provider for 5.0 will be less than was required to update for 3.0.
+
+## Themes
+
+We have extracted a few major areas or themes which will form the basis for the large investments in EF Core 5.0.
+
+## Many-to-many navigation properties (a.k.a "skip navigations")
+
+Lead developers: @smitpatel and @AndriySvyryd
+
+Tracked by [#19003](https://github.com/aspnet/EntityFrameworkCore/issues/19003)
+
+T-shirt size: L
+
+Many-to-many is the most requested feature (~407 votes) on the GitHub backlog. Support for many-to-many relationships can be broken down into three major areas:
+
+* Skip navigation properties. These allow the model to be used for queries, etc. without reference to the underlying join table entity.
+* Property-bag entity types. These allow a standard CLR type (e.g. `Dictionary`) to be used for entity instances such that an explicit CLR type is not needed for each entity type.
+* Sugar for easy configuration of many-to-many relationships.
+
+We believe that the most significant blocker for those wanting many-to-many support is not being able to use the "natural" relationships, without referring to the join table, in business logic such as queries. The join table entity type may still exist, but it should not get in the way of business logic. This is why we have chosen to tackle skip navigation properties for 5.0.
+
+At this time the other parts of many-to-many are being pursued as a stretch goal for EF Core 5.0. This means they are not currently in the plan for 5.0, but if things go well we hope to pull them in.
+
+## Table-per-type (TPT) inheritance mapping
+
+Lead developer: @AndriySvyryd
+
+Tracked by [#2266](https://github.com/aspnet/EntityFrameworkCore/issues/2266)
+
+T-shirt size: XL
+
+We're doing TPT because it is both a highly requested feature (~254 votes; 3rd overall) and because it requires some low-level changes that we feel are appropriate for the foundational nature of the overall .NET Core 5.0 plan. We expect this to result in breaking changes for database providers, although these should be much less severe than the changes required for 3.0.
```

Comment:
```suggestion
We're doing TPT because it is both a highly requested feature (~254 votes; 3rd overall) and because it requires some low-level changes that we feel are appropriate for the foundational nature of the overall .NET 5.0 plan. We expect this to result in breaking changes for database providers, although these should be much less severe than the changes required for 3.0.
```